config.rb

```diff
@@ -92,6 +91,9 @@ def guides_by_chapter
 # Development-specific configuration
 configure :development do
   activate :livereload
+  activate :disqus do |d|
```

Comment:
Other plugins use hash style configuration rather than block style configuration (`activate :disqus, shortname: nil`). Possible to update both `activate`s to this style?